src/xdg-desktop-portal-gtk.c

```diff
@@ -130,6 +131,12 @@ on_bus_acquired (GDBusConnection *connection,
       g_warning ("error: %s\n", error->message);
       g_clear_error (&error);
     }
+
+  if (!email_init (connection, &error))
+    {
+      g_warning ("error: %s\n", error->message);
+      g_clear_error (&error);
+    }
```

Comment:
Similar comment than https://github.com/flatpak/xdg-desktop-portal/pull/86/commits/32550005522e38b5bba98b26d10dc5ff455af281#r101017048: not for now, but looks like this function could use some refactoring too